src/containers/Pages/Job/page/JobList/JobList.test.tsx

```diff
@@ -0,0 +1,19 @@
+import {render} from "@testing-library/react";
+import JobList from './JobList';
+import React from "react";
+
+import {store} from '../../../../../store/store';
+import {Provider} from 'react-redux';
+
+test('render JobList components', () => {
+
+    const { getByText } = render(
+        <Provider store={store}>
+            <JobList />
+        </Provider>
+    );
+    
+    const linkElement = getByText(/IT jobs are available/i);
```

Comment:
The test scenario should be
- render component
- the component will call the API and get the list of jobs (e.g: 1234 jobs)
- expect `1234 IT jobs are available` is in the document

But this is the unit test, so we will call a mock API
Please refer: https://jestjs.io/docs/en/manual-mocks

Reply:
**JobList** is for Task02 (display first 100 jobs) but I wrote it while you're reviewing my previous PR.
So I think **JobList** should be dump component to prevent the abuse of redux (_which is messy and
not necessary in this case_.) and make it easy to test. 

**_So what I have done:_**

- Recode **JobList** to dump components, so **JobList** now only recieve props from parent component.
- The same with **JobHeader**.

Sorry for the late response, I just got well not long ago since I got sick ❤